package.json

```diff
@@ -39,7 +39,7 @@
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^3.0.0",
-    "ganache-cli": "^6.12.2",
+    "ganache-cli": "^6.10.2",
```

Comment:
Why downgrade?

Reply:
This is already outdated so resolving this issue.